backport_packages/setup_backport_packages.py

```diff
@@ -243,10 +257,18 @@ def _remover(node: LN, capture: Capture, filename: Filename) -> None:
         .modify(add_provide_context_to_python_operator)
     )
 
+    # Remove new class
     remove_class(qry, "GKEStartPodOperator")
 
     qry.execute(write=True, silent=False, interactive=False)
 
+    # Add old import to GKE
+    gke_path = os.path.join(
+        dirname(__file__), "airflow", "providers", "google", "cloud", "operators", "kubernetes_engine.py"
+    )
+    with open(gke_path, "a") as f:
+        f.writelines(["", "from airflow.contrib.operators.gcp_container_operator import GKEPodOperator"])
```

Comment:
Should we remove this file?
https://github.com/apache/airflow/blob/50ee275a4090063df6b91cd9162dae32f8cbc4e0/airflow/providers/google/cloud/example_dags/example_kubernetes_engine_setup.py

Reply:
Probably yes, I will check if the test works...

Reply:
Works out of the box 🎂